packages/experiments/src/components/Chiclet/ChicletCard.styles.ts

```diff
@@ -22,7 +23,8 @@ export const getStyles = (props: IChicletCardStyleProps): IChicletCardStyles =>
             cursor: 'pointer'
           }
         }
-      }
+      },
+      className
```

Comment:
@natalieethell Note that by passing in the className and mixing it at the end of "root", `merge-styles` can merge the classname correctly. If you pass a `mergeStyles` generated classname into the child, the string can be dereferenced into an object, and can be mixed together correctly resulting in no specificity wars.

Reply:
Cool, thanks!